src/mongo/models.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+
+const BRIDGE_CLAIM = 'bridgeClaim';
```

Comment:
last one. shorter and we know it's related to bridge already:
```suggestion
const BRIDGE_CLAIM = 'claims';
```